master/internal/root.go

```diff
@@ -17,17 +18,17 @@ func root(ctx *actor.Context) error {
 	case actor.ChildFailed:
 		switch msg.Child.Address() {
 		case sproto.K8sRMAddr, sproto.AgentRMAddr:
-			ctx.Log().WithField("crash", msg).Errorf(clusterCrashMessage)
+			return errors.New(clusterCrashMessage)
```

Comment:
let's log the `msg` still, in all cases (should have a useful message, at least for `actor.ChildFailed`). if you noticed the system already prints it (cant remember if it does or not), then this is fine.

Reply:
system does print it to log for devcluster -- should I still log?

Reply:
I just read all the actor code, it should all be logged already, nvm. Kinda strange to go against the 'handle an error exactly once' thing. Then you need to read implementations to prevent double logs or missing logs.